src/common/state/redis.c

```diff
@@ -54,21 +55,26 @@ db_handle *db_connect(const char *address,
   db_handle *db = malloc(sizeof(db_handle));
   /* Sync connection for initial handshake */
   redisReply *reply;
-  long long num_clients;
   redisContext *context = redisConnect(address, port);
   CHECK_REDIS_CONNECT(redisContext, context, "could not connect to redis %s:%d",
                       address, port);
   /* Add new client using optimistic locking. */
+  db_client_id client = globally_unique_id();
   while (1) {
```

Comment:
Yeah. I'm not feeling strongly about it, so feel free to keep as-is if you want.